examples/wc-wallet-react/src/App.tsx

```diff
@@ -23,7 +23,9 @@ export default function App(): any {
   }, [walletConnectCtx.sessions])
 
   useEffect(() => {
-    walletConnectCtx.setAdapter(new WalletConnectNeonAdapter(accountCtx.rpcAddress, accountCtx.account))
+    walletConnectCtx.setAdapters({
+      neo3: new WalletConnectNeonAdapter(accountCtx.rpcAddress, accountCtx.account),
+    })
```

Comment:
this is going on the TOptions now, right? I am a bit confused.

Reply:
No, because you need to change the account when the accountCtx.account change